azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/launcher/CallingCompositeJavaLauncher.java

```diff
@@ -32,11 +35,14 @@ public void launch(final CallLauncherActivity callLauncherActivity,
                        final UUID groupId,
                        final String meetingLink,
                        final Function1<? super String, Unit> showAlert) {
-
+        final Map<String, String> overrideString = new HashMap<>();
```

Comment:
do we want to leave this in?

Reply:
No. this is just for testing. If anyone wants to checkout to this branch and run the app to see the change. 
I will remove it before making the PR for merge.

Reply:
Please put this behind a feature flag (in the demo) so that it can be enabled/disabled to demonstrate public API, it should be off by default, but easily enabled for a tester or someone who wants to inspect the api.

Reply:
I.e. "Demonstrate Custom Strings" feature.

Reply:
This has been removed.